tenets/codelingo/code-review-comments/declare-empty-slice/codelingo.yaml

```diff
@@ -51,4 +53,4 @@ tenets:
           end_offset
           go.composite_lit:
             child_count == 1
-            go.array_type
+            go.array_type
```

Comment:
Probably better to have a newline at the end of file - but it's not a big deal either way.